requirements.txt

```diff
@@ -40,3 +40,5 @@ google-jetstream@git+https://github.com/AI-Hypercomputer/JetStream.git
 jsonlines
 pathwaysutils@git+https://github.com/AI-Hypercomputer/pathways-utils.git
 omegaconf
+torch==2.4.1
+safetensors==0.4.5
```

Comment:
This will make MaxText install size much larger.

let's not add those dependancies in `requirements.txt` we can add an instruction to install them before running your script, or we can make your script a bash that installs those.

Reply:
Done